frontends/web/src/contexts/LocalizationContext.ts

```diff
@@ -0,0 +1,11 @@
+import { createContext } from 'react';
+
+type DarkModeContextProps = {
+    decimal: string;
+    group: string;
+  }
+
+
+// Create a Context for the localization values
+const LocalizationContext = createContext<DarkModeContextProps>({}as DarkModeContextProps);
+export default LocalizationContext;
```

Comment:
DarkModeContextProps -> LocalizationContextProps

nit: I think it's better to prefer named exports if possible